packages/slice-machine/components/ScreenshotChangesModal/useCustomScreenshot.tsx

```diff
@@ -30,6 +30,7 @@ const FileInputRenderer: React.FC<HandleFileProp> = ({
         justifyContent: "center",
         width: 200,
         alignItems: "center",
+        flex: 1,
```

Comment:
I haven't tested this locally, but doesn't this make the button take up the remaining flex space and therefore be 50/50 sharing space with the open simulator button? Because the designs aren't like that I think
<img width="581" alt="image" src="https://user-images.githubusercontent.com/47107427/199056360-771d7a39-90ca-4228-a685-efcb5e3b6009.png">

Reply:
It does takes the remaining space but they're not 50/50.
I verified the design and that part also so the capture screenshot button wouldn't change size with or without the other button :)